docs-sources/includes/_Templates.md

```diff
@@ -0,0 +1,97 @@
+# Templates
```

Comment:
@ronkorving @almirkadric I will be using this description as a spec for how meta-package templates will be structured and how they will work. Please make sure to add as many comments and suggestions here as possible.